src/main/kotlin/io/zerobase/smarttracing/features/devices/DevicesApiFeatureFactory.kt

```diff
@@ -0,0 +1,17 @@
+package io.zerobase.smarttracing.features.devices
+
+import com.fasterxml.jackson.annotation.JsonTypeName
+import com.google.inject.Injector
+import io.dropwizard.setup.Environment
+import io.zerobase.smarttracing.config.AppConfig
+import io.zerobase.smarttracing.features.FeatureFactory
+
+@JsonTypeName("devices")
+class DevicesApiFeatureFactory(override val enabled: Boolean): FeatureFactory {
+    override fun build(env: Environment, config: AppConfig, injector: Injector) {
+        if (enabled) {
+            env.jersey().register(DevicesResource(injector.getInstance(DevicesDao::class.java)))
+            env.jersey().register(UsersResource(injector.getInstance(UsersDao::class.java)))
```

Comment:
why are users grouped together with devices resource?

Reply:
mostly because, at the moment, a "user" is nothing more than contact information attached to a device. if users become something more, we can split it out to it's own domain or reverse the relationship so the domain is users and devices are grouped with them